jerry-core/parser/js/js-parser-expr.c

```diff
@@ -45,12 +45,17 @@
 /**
  * Precedence for exponentiation operation.
  */
-#define PARSER_RIGHT_TO_LEFT_ORDER_EXPONENTIATION 15
+#define PARSER_RIGHT_TO_LEFT_ORDER_EXPONENTIATION 16
 
 /**
  * Value of grouping level increase and decrease.
  */
-#define PARSER_GROUPING_LEVEL_INCREASE 2
+#define PARSER_GROUPING_LEVEL_INCREASE 4
+
+/**
+ * Value for testing logical opeartor found.
```

Comment:
`Represents whether logical expression was emitted in the current group expression.`